modules/dynamic-plugins/ref-rh-tech-preview-plugins.adoc

```diff
@@ -1,4 +1,297 @@
+|`./dynamic-plugins/dist/pagerduty-backstage-plugin-backend-dynamic`
+
+`PAGERDUTY_API_BASE`
+
+`PAGERDUTY_CLIENT_ID`
+
+`PAGERDUTY_CLIENT_SECRET`
+
+`PAGERDUTY_SUBDOMAIN`
+
+
+|Pingidentity  |`https://npmjs.com/package/@backstage-community/plugin-catalog-backend-module-pingidentity/v/0.1.5[@backstage-community/plugin-catalog-backend-module-pingidentity]` |0.1.5 
+|`./dynamic-plugins/dist/backstage-community-plugin-catalog-backend-module-pingidentity-dynamic`
+
+
+|Scaffolder Relation Processor  |`https://npmjs.com/package/@backstage-community/plugin-catalog-backend-module-scaffolder-relation-processor/v/2.0.2[@backstage-community/plugin-catalog-backend-module-scaffolder-relation-processor]` |2.0.2 
+|`./dynamic-plugins/dist/backstage-community-plugin-catalog-backend-module-scaffolder-relation-processor-dynamic`
+
+
+|Security Insights  |`https://npmjs.com/package/@roadiehq/backstage-plugin-security-insights/v/2.4.0[@roadiehq/backstage-plugin-security-insights]` |2.4.0 
+|`./dynamic-plugins/dist/roadiehq-backstage-plugin-security-insights`
+
+
+|ServiceNow  |`https://npmjs.com/package/@backstage-community/plugin-scaffolder-backend-module-servicenow/v/2.2.3[@backstage-community/plugin-scaffolder-backend-module-servicenow]` |2.2.3 
+|`./dynamic-plugins/dist/backstage-community-plugin-scaffolder-backend-module-servicenow-dynamic`
+
+`SERVICENOW_BASE_URL`
+
+`SERVICENOW_USERNAME`
+
+`SERVICENOW_PASSWORD`
+
+
+|Signals  |`https://npmjs.com/package/@backstage/plugin-signals/v/0.0.11[@backstage/plugin-signals]` |0.0.11 
+|`./dynamic-plugins/dist/backstage-plugin-signals`
+
+
+|SonarQube  |`https://npmjs.com/package/@backstage-community/plugin-sonarqube/v/0.8.7[@backstage-community/plugin-sonarqube]` |0.8.7 
+|`./dynamic-plugins/dist/backstage-community-plugin-sonarqube`
+
+
+|SonarQube  |`https://npmjs.com/package/@backstage-community/plugin-sonarqube-backend/v/0.3.0[@backstage-community/plugin-sonarqube-backend]` |0.3.0 
+|`./dynamic-plugins/dist/backstage-community-plugin-sonarqube-backend-dynamic`
+
+`SONARQUBE_URL`
+
+`SONARQUBE_TOKEN`
+
+
+|SonarQube  |`https://npmjs.com/package/@backstage-community/plugin-scaffolder-backend-module-sonarqube/v/2.2.2[@backstage-community/plugin-scaffolder-backend-module-sonarqube]` |2.2.2 
+|`./dynamic-plugins/dist/backstage-community-plugin-scaffolder-backend-module-sonarqube-dynamic`
+
+
+|Tech Radar  |`https://npmjs.com/package/@backstage-community/plugin-tech-radar/v/1.0.0[@backstage-community/plugin-tech-radar]` |1.0.0 
+|`./dynamic-plugins/dist/backstage-community-plugin-tech-radar`
+
+
+|Tech Radar  |`https://npmjs.com/package/@backstage-community/plugin-tech-radar-backend/v/1.0.0[@backstage-community/plugin-tech-radar-backend]` |1.0.0 
+|`./dynamic-plugins/dist/backstage-community-plugin-tech-radar-backend-dynamic`
+
+`TECH_RADAR_DATA_URL`
+
+
+|Topology  |`https://npmjs.com/package/@backstage-community/plugin-topology/v/1.29.7[@backstage-community/plugin-topology]` |1.29.7 
```

Comment:
@nickboldt Topology was a fully supported plugin in RHDH 1.3, has it's status changed in RHDH 1.4?

Reply:
fixed in latest update to this PR

Reply:
Fixed.

New counts:

* 18 RH GA plugins
* 54 RH TP plugins
* 0 Community plugins